Common/Source/Dialogs/dlgRadioSettings.cpp

```diff
@@ -154,48 +157,56 @@ int Idx=0;
   {
     RadioPara.Changed =FALSE;
     TCHAR Name[250];
-    if(_tcscmp(_T("        "), RadioPara.ActiveName ) == 0)
+    if((_tcscmp(_T("        "), RadioPara.ActiveName ) == 0)  || ( _tcslen(RadioPara.ActiveName) == 0))
       Idx = SearchStation(RadioPara.ActiveFrequency);
     if(Idx !=0)
     {
-        _stprintf(RadioPara.ActiveName,_T("%s"),WayPointList[Idx].Name);
+
+    		CopyTruncateString(RadioPara.ActiveName, NAME_SIZE, WayPointList[Idx].Name);
```

Comment:
can be removed ?

RadioPara.ActiveName will be overwrited by devPutFreqActive call line 170

Reply:
ok changed